### DISCUSSION TITLE
Collaboration between Novatec Consulting GmbH and envite consulting GmbH

### DISCUSSION THREAD

**Discussion by mustaphazorgati** [Taskana/taskana#2065] [Announcements]
The TASKANA Team welcomes envite consulting GmbH as a collaboration partner!

As discussed we want to transparently document our collaboration. This can be expanded at any time, if needed.

* The existing [open source backlog](https://taskana.atlassian.net/jira/software/c/projects/TSK/boards/1) is used for organization and prioritization. 
* The [collaboration model from the TASKANA team](https://taskana.atlassian.net/wiki/spaces/TAS/pages/42860556/Contributor+Guide) remains in place. 
* Working dependency bumps can be done by any party at any time. 
* In the case of "significant" (technical) decisions, the other party is involved and a solution acceptable to both parties is sought. 
    Preferably, a direct discussion is sought, whereby the joint decision from the discussion is recorded in some form in writing. This ensures that one party does not subsequently change its mind.
* Pull requests are - as a rule - reviewed by the respective party. Exception: Something else is discussed for a specific ticket.
* Representatives of each party meet every week (15 - 20 min) and discuss the current situation, so that both parties are always up-to-date.
   * Among other things, part of the meeting is the distribution of general maintenance obligations (e.g. dependency bumps, where something has to be done manually).
   * Part of the meeting is the arrangement of an additional review from the other party for selected tickets.
* A release is only performed after mutual agreement. This agreement can be made during the regular meeting. Alternatively, because e.g. something must be published promptly, the arrangement can take place e.g. telephonically. Both parties should remain capable of acting and thus be able to independently publish a release (after consultation).

For the sake of documentation: Please comment that you're in agreement to these collaboration terms :) Thanks!

**Comment by mustaphazorgati** [comment]
I agree to those collaboration terms

**Comment by holgerhagen** [comment]
I do agree as well.

**Comment by gitgoodjhe** [comment]
I also agree
